service/src/main/java/io/camunda/service/CamundaServices.java

```diff
@@ -65,6 +66,11 @@ public MessageServices messageServices() {
     return new MessageServices(brokerClient, searchClient, transformers, authentication);
   }
 
+  public DocumentServices documentServices() {
+    return new DocumentServices(
+        brokerClient, searchClient, transformers, authentication, new HashMap<>());
```

Comment:
❓ What is the benefit of initializing the documents with a `HashMap` here? I would argue that the document services should take care of this. Here in the config, we don't know how the documents are stored/handled/cached and shouldn't need to know. Either the document services do this themselves or inject a `documentStore` (which currently is implemented using a simple in-memory map).

Reply:
Right, no benefit here - I concealed this inside the implementation